### DISCUSSION TITLE
[잡담] 개인 repository 생성해서 과제나 코드 연습에 사용할 예정입니다.

### DISCUSSION THREAD

**Discussion by ghost** [4Moyede/HexatonClass-01#81] [잡담]
깃허브에 좀 더 친숙해지기 위해 제 개인 repository를 보다 능동적으로 사용해 볼 생각입니다.

과제 파일을 이곳에도 push하고 제 개인 repository에도 동시에 push해서 깃허브의 시스템에 대해 익히고

나아가서는 여러분 뿐만 아니라 다른 사람들과 코드 짜는 여러 기술에 대해 이야기를 주고받고 싶습니다.

확실히 저번에 개고생 하고 나니까 뭘 만져도 해낼 수 있다는 자신감이 붙었습니다.

**Comment by dev-LeoKang** [comment]
repository URL 공유해주세요!

**Reply by ghost** [reply]
https://github.com/JaebeomHan/Practice 입니다~~
